Raven.Tests.Core/BulkInsert/ChunkedBulkInsert.cs

```diff
@@ -53,13 +46,15 @@ public void ChunkVolumeConstraint()
                             Children = Enumerable.Range(0, 5).Select(x => new Node { Name = "Child" + x }).ToArray()
                         });
                     }
+                    bulkInsertStartsCounter = ((ChunkedRemoteBulkInsertOperation)bulkInsert.Operation).RemoteBulkInsertOperationSwitches;
                 }
 
                 Assert.Equal(20, bulkInsertStartsCounter);
                 using (var session = store.OpenSession())
                 {
                     var count = session.Query<Node>().Customize(x => x.WaitForNonStaleResults()).Count();
-                    Assert.Equal(20, count);
+                    if (20 != count)
```

Comment:
You aren't actually checking anything in here.

Reply:
Fixed